aws/table_aws_sfn_state_machine_execution.go

```diff
@@ -48,7 +46,7 @@ func tableAwsStepFunctionsStateMachineExecution(_ context.Context) *plugin.Table
 			},
 			{
 				Name:        "status",
-				Description: "The current status of the execution.",
+				Description: "The current status of the execution (valid values are `RUNNING`, `SUCCEEDED`, `FAILED`, `TIMED_OUT`, `ABORTED`).",
```

Comment:
```suggestion
				Description: "The current status of the execution (RUNNING | SUCCEEDED | FAILED | TIMED_OUT | ABORTED).",
```

AWS uses this format when listing valid values in some other descriptions, so we could stay consistent here even though the official AWS API docs don't have them listed this way